spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -205,3 +213,53 @@ This approach is depicted in the following Figure that shows an overview of the
+For clarity, we use `Hp*` and `Hc*` to denote block heights on the provider chain and consumer chain, respectively. 
+
+![Mapping Between Provider and Consumer Heights](./figures/ccv-height-mapping-overview.png?raw=true)
+
+- For every block, the provider CCV module maps the ID of the VSC it provides to the consumer chains to the height of the subsequent block, i.e., `VSCtoH(VSC.id) = Hp + 1`, for a VSC provided at height `Hp`. 
+  Intuitively, this means that the validator updates in a provided VSC will update the voting power at height `VSCtoH(VSC.id)`.
+- For every block, every consumer CCV module maps the height of the subsequent block to the ID of the latest received VSC, e.g., `HtoVSC(Hc2 + 1) = VSC1.id`. 
+  Intuitively, this means that the voting power on the consumer chain during a block `Hc` was updated by the VSC with ID `HtoVSC(Hc)`.
+  > **Note**: It is possible for multiple VSCs to be received by the consumer chain within the same block. For more details, take a look at the [Validator sets, validator updates and VSCs](./system_model_and_properties.md#validator-sets-validator-updates-and-vscs) section.
+- By default, every consumer CCV module maps any block height to `0` (i.e., VSC IDs start from `1`). 
+  Intuitively, this means that the voting power on the consumer chain at height `Hc` with `HtoVSC(Hc) = 0` was setup at genesis during Channel Initialization. 
+- For every consumer chain, the provider CCV module sets `VSCtoH(0)` to the height at which the first VSC was provided to this consumer chain. 
+  Intuitively, this means that the validator set on the provider chain at height `VSCtoH(0)` matches the validator set on the consumer chain at all heights `Hc` with `HtoVSC(Hc) = 0`.
+
+The following figure shows an overview of the Consumer Initiated Slashing operation of CCV. 
+
+![Consumer Initiated Slashing](./figures/ccv-evidence-overview.png?raw=true)
+
+- At height `Hc2`, the consumer chain receives evidence that a validator `V` misbehaved at height `Hc1`. 
+  As a result, the consumer CCV module sends a `SlashPacket` to the provider chain: 
+  It makes a request to slash `V`, but it replaces the infraction height `Hc1` with `HtoVSC(Hc1)`, 
+  i.e., the ID of the VSC that updated the "misbehaving voting power" or `0` if such a VSC does not exist.
+- The provider CCV module receives at height `Hp1` the `SlashPacket` with `vscId = HtoVSC(Hc1)`. 
+  As a result, it requests the provider Slashing module to slash `V`, but it set the infraction height to `VSCtoH(vscId)`, i.e., 
+    - if `vscId != 0`, the height on the provider chain where the voting power was updated by the VSC with ID `vscId`;
+    - otherwise, the height at which the first VSC was provided to this consumer chain.
+  > **Note**: As a consequence of slashing (and potentially jailing) `V`, the Staking module updates accordingly `V`'s voting power. This update MUST be visible in the next VSC provided to the consumer chains.  
```

Comment:
Is this MUST requirement enforced anywhere or is it simply a clarification for the reader?

Reply:
Here is just a not for the reader, but it's actually an assumption of the ABCI application (i.e., the state machine). The test case should be in the Staking module of Cosmos SDK (i.e., if during a block the voting power changes due to a Slash operation, at the end of the block, that change must be mirrored in the list of updates provided to the underlying consensus engine).